core/shared/src/main/scala/org/polyvariant/colorize.scala

```diff
@@ -121,12 +150,40 @@ object colorize {
 
   private[colorize] case class CurrentColor(value: String) extends AnyVal
 
+  sealed trait ColorMode
+
+  object ColorMode {
+    private[colorize] case object Ansi extends ColorMode
+    private[colorize] case object TrueColor extends ColorMode
+  }
+
   final case class RenderConfig(
-    resetString: String
+    mode: ColorMode,
```

Comment:
mima is definitely not happy about this change

Reply:
that's alright